### ISSUE TITLE
Get-ComputerInfo incorrectly implemented

### ISSUE THREAD

**Issue by comnam90** [PowerShell/PrivateCloud.DiagnosticInfo#131]
Hi guys,

It seems that in response to issue #123, a pull request #126 was created to introduce the use of `Get-ComputerInfo`.
This is a great addition to the data collection, however it was implemented incorrectly and as such isn't actually working.
![Image of error](https://i.imgur.com/ySXNlFs.png)

Currently it is trying to use the `-CimSession` parameter, but this isn't a supported parameter and instead should be executed via `Invoke-Command` to be run remotely when collecting the information.
```
Invoke-Command -ComputerName _C_ -ScriptBlock { Get-ComputerInfo }
```

Cheers,
Ben

**Comment by jlgandy2** [comment]
Yes I agree with you comname90. Get-ComputerInfo does NOT have a property of -cimsession. See this link. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.management/get-computerinfo?view=powershell-6

Proposed change to line 1939:
                                'Get-ComputerInfo',

**Comment by jlgandy2** [comment]
Any update on this?Any update on this?

**Comment by jlgandy2** [comment]
Its been a long time guys. Any update on this?
